app/src/main/java/ai/elimu/appstore/BaseApplication.java

```diff
@@ -90,9 +94,15 @@ public Retrofit getRetrofit() {
                 .retryOnConnectionFailure(true)
                 .build();
 
+        /**
+         * Gson object used in convert factory for serialization and deserialization of objects
+         */
+        Gson gson = new GsonBuilder().setLenient().create();
```

Comment:
What's the difference between this and `new Gson();`?

https://github.com/elimu-ai/appstore/blob/91e58f588e1ceacf9a8b95e60626ea76b5ceea2b/app/src/main/java/ai/elimu/appstore/synchronization/AppSynchronizationActivity.java#L132